### PR TITLE
Remove arg check from docker-options/functions, global var cleanup

### LINKED ISSUE #1869: docker-options/functions has an arg check
docker-options/functions has an arg check which is preventing other plugins from sourcing it as it causes an exit.

```bash
[[ " docker-options docker-options:add docker-options:remove help docker-options:help " == *" $1 "* ]] || exit "$DOKKU_NOT_IMPLEMENTED_EXIT"
```

I git blame @michaelshobbs for not catching that :yum:

### PR DESCRIPTION
**Pull request by u2mejc**
Removes another global var and resolves #1869

### CODE REVIEW

File: plugins/docker-options/functions [hunk 1]
```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env bash
-[[ " docker-options docker-options:add docker-options:remove help docker-options:help " == *" $1 "* ]] || exit "$DOKKU_NOT_IMPLEMENTED_EXIT"
```
Comment (michaelshobbs) [inline plugins/docker-options/functions]: this will need to move to `commands`

Comment (u2mejc) [inline plugins/docker-options/functions]: agreed, one sec

File: plugins/docker-options/commands [hunk 2]
```diff
@@ -4,6 +4,8 @@ set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
 source "$PLUGIN_CORE_AVAILABLE_PATH/common/functions"
```
Comment (u2mejc) [inline plugins/docker-options/commands]: Look at line two above here... :smile:

Comment (michaelshobbs) [inline plugins/docker-options/commands]: ah!! github folding!!!!!

**Comment by u2mejc** [comment]
This fixes a bug I introduced in the 0.4.x branch, and passed circle ci, I'm going to merge.

**Comment by josegonzalez** [comment]
I'll make another release so that 0.4.x doesn't have an unstable final release.

**Comment by u2mejc** [comment]
:+1:

**Pull request merged**